components/advocates/MeetTheAdvocates.vue

```diff
@@ -57,55 +57,45 @@
   </section>
 </template>
 
-<script lang="ts">
-import Vue from 'vue'
+<script setup lang="ts">
 import { mapState } from 'vuex'
 import type { MapperForStateWithNamespace } from 'vuex'
-import { Component, Prop } from 'vue-property-decorator'
 import { Advocate, ADVOCATES_WORLD_REGION_OPTIONS, State } from '~/store/advocates'
 
-@Component({
-  computed: {
-    ...mapState<MapperForStateWithNamespace>('advocates', {
-      regionFilters: (state: State) => state.regionFilters
-    })
-  }
-})
-export default class MeetTheAdvocates extends Vue {
-  @Prop(Array) advocates!: Advocate[]
+interface Props {
+  advocates: Advocate[]
+}
 
-  /**
-   * Region filters from Vuex store.
-   *
-   * Initialized with mapState.
-   */
-  public regionFilters!: string[]
+defineProps<Props>()
 
-  private filter = {
-    label: 'Locations',
-    options: ADVOCATES_WORLD_REGION_OPTIONS
-  }
+const filter = {
+  label: 'Locations',
+  options: ADVOCATES_WORLD_REGION_OPTIONS
+}
 
-  isRegionFilterChecked (filterValue: string): boolean {
-    return this.regionFilters.includes(filterValue)
-  }
+const { regionFilters } = mapState<MapperForStateWithNamespace>('advocates', {
+  regionFilters: (state: State) => state.regionFilters
+})
 
-  updateRegionFilter (option: string, isChecked: boolean): void {
-    const regionFilters = this.regionFilters.filter(oldOption => oldOption !== option)
+const isRegionFilterChecked = (filterValue: string): boolean => {
+  return regionFilters.includes(filterValue)
+}
 
-    if (isChecked) {
-      regionFilters.push(option)
-    }
+const updateRegionFilter = (option: string, isChecked: boolean): void => {
+  const filteredRegionFilters = regionFilters.filter(oldOption => oldOption !== option)
 
-    this.updateRegionFilters(regionFilters)
+  if (isChecked) {
+    filteredRegionFilters.push(option)
   }
 
-  updateRegionFilters (regionFilters: string[]): void {
-    this.$store.commit('advocates/setRegionFilters', regionFilters)
-  }
+  updateRegionFilters(filteredRegionFilters)
+}
 
-  joinSlackLink: string = 'https://ibm.co/joinqiskitslack'
+const updateRegionFilters = (regionFilters: string[]): void => {
```

Comment:
Same for `void`

Reply:
Changed in https://github.com/Qiskit/qiskit.org/pull/2935/commits/9d5068d2ff914d97698ea3b0b2b63d6c50d686dc.